src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -304,17 +308,35 @@ private void ImportBreak()
         private void ImportLoadVar(int index, bool argument)
         {
             int varBase;
+            int varCountBase;
             int varOffset;
             LLVMTypeRef valueType;
             TypeDesc type;
 
             if (argument)
             {
+                varCountBase = 0;
                 varBase = 0;
-                // todo: this is off by one for instance methods
+                if(!_signature.IsStatic)
```

Comment:
Would be nice to do search and replace for all occurrences of `if(` in this file. We follow CoreFX coding guidelines when it comes to whitespaces.